packages/app-store/wipemycalother/lib/reschedule.ts

```diff
@@ -30,6 +32,16 @@ const Reschedule = async (bookingUid: string, cancellationReason: string) => {
       location: true,
       attendees: true,
       references: true,
+      eventType: {
+        include: {
```

Comment:
Do we need everything in the eventType? or why are we not using select in here? @Udit-takkar

Reply:
we don't need other event type fields